### PR TITLE
enforce write only access to folder http endpoint

### PR DESCRIPTION
**Pull request by jlewitt1**

### CODE REVIEW

**Comment by jlewitt1** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/run-house/runhouse/1109?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1109** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/1109?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1108** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/1108?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jlewitt1 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: runhouse/servers/http/auth.py [hunk 1]
```diff
@@ -75,6 +75,7 @@ def clear_cache(self, token: str = None):
 async def averify_cluster_access(
     cluster_uri: str,
     token: str,
+    write_only_access: bool = False,
```
Comment (rohinb2) [inline runhouse/servers/http/auth.py:78]: instead of a bool here, can just do `access_level_required` and pass `ResourceAccess.WRITE`

Comment (jlewitt1) [reply runhouse/servers/http/auth.py:78]: yea good call

**Comment by jlewitt1** [comment]
### Merge activity

* **Aug 12, 4:15 PM EDT**: @jlewitt1 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/run-house/runhouse/1109).

**Pull request merged**